### DISCUSSION TITLE
Add " tools: " Prefix

### DISCUSSION THREAD

**Discussion by karim-eg** [Sketchware-Pro/Sketchware-Pro#454] [Q&A]
**When I try to add tools: prefix to my project, sketchware didn't recognize it and Says "error: unbound prefix".**

I wanna use it at permission or activity at manifest file such as:

```
<uses-permission android:name="android.permission.MANAGE_EXTERNAL_STORAGE" tools:ignore="ScopedStorage" />
```

**Comment by j-px** [comment]
This is purely an error in the code snipped you added. Remove `tools:ignore="ScopedStorage" ` from what you added, that fixes your issue.

Since Sketchware Pro doesn't have a linter, the ignore attribute isn't needed.

**Reply by karim-eg** [reply]
Okay but I don't mean only this line tools:ignore="ScopedStorage"

**I mean that Prefix called tools: 
Isn't supported at Sketchware Pro**

**Reply by j-px** [reply]
It isn't "supported" unless you register the prefix yourself. For example, at the start of the manifest: `<manifest … xmlns:tools="http://schemas.android.com/tools">`

**Reply by karim-eg** [reply]
**Yeah got it thanks** ♥️♥️
